event/consumer_test.go

```diff
@@ -28,6 +28,7 @@ var (
 		Summary:         "",
 		ReleaseDate:     "",
 		Title:           "testTitle1",
+		Topics:          []string{""},
```

Comment:
For both scenario of slice size 0 or 1 it will work. Also in 3rd scenario, incase topic element is missing (null) , should be validated in the calling service to pass on a valid slice. Otherwise, the schema definition will throws a valid error message while unmarshalling, if an element is missing in the incoming event.

Reply:
Justins suggestion will work once we have fixed the avro package to handle empty arrays, for now can we create 2 tests, one with empty array and one where the array has an empty string.

Then comment out the test with empty array with TODO comment saying to reinstate when fix has been applied to dp-kafka avro package

Reply:
as discussed, three possible additional test scenarios added.